lib/app.js

```diff
@@ -23,7 +23,7 @@ var defaultSettings = {
 
 function getPrompt(argv, schema, onComplete, index, results) {
   if (!Array.isArray(schema)) {
-    console.log(chalk.red('An error occurred'));
+    console.log(chalk.red('getPrompt schema must be an Array.'));
```

Comment:
Ok, that's a fair point. Lets leave it as what you have here - we need a complete messaging overhaul soon